CHANGELOG.md

```diff
@@ -1,5 +1,54 @@
 Change Log
 ==========
+
+# Version 3.3.0
+
+_2022-05-04_
+
+This release brings WebSocket related improvements, support for HMPP, and a some performance improvements and bug fixes!
+
+## ✨️ [new] `WebSocketNetworkTransport.closeConnection` (#4049)
+
+This new method can be used in conjunction with [`reopenWhen`](https://apollographql.github.io/apollo-kotlin/kdoc/apollo-runtime/com.apollographql.apollo3/-apollo-client/-builder/web-socket-reopen-when.html)  to force a reconnection to the server. This could be useful for instance when needing to pass new auth tokens in the headers.
```

Comment:
Mention the link to 2.x `reconnect`? 

```
If you were using `subscriptionManager.reconnect()` in 2.x, `closeConnection` is a simple way to achieve the same behaviour.
```

And update the migration guide?

Reply:
(migration guide already pending [here](https://github.com/apollographql/apollo-kotlin/pull/4052/files?w=1))